src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/VoteOutcome/VoteResults/VoteResults.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { defineMessages } from 'react-intl';
 
-import { MotionData } from '~gql';
+import { MotionStakes, VoterRecord } from '~gql';
```

Comment:
Could these be either derived from `MotionData` or have fragments defined? Right now they describe the object type as defined in the schema rather than what we're querying for. If there's a new field in the schema, it will be included in this type regardless of whether it's returned in the query.

Reply:
For ongoing development, yes. However, I can see the potential this has for giving us headache when refactoring. This was one of the reasons for moving away from Amplify OOTB codegen which would generate types for all the fields as opposed to what we query for

Reply:
I'll see if we can use fragments instead then